### ISSUE TITLE
Problems with reproducing results.

### ISSUE THREAD

**Issue by Parskatt** [qianqianwang68/caps#4]
Hi,

I'm having difficulties reproducing the results of the pretrained model on HPatches.
While the pretrained model performs well, training by myself results in no improvements on HPatches compared to a simple ImageNet init (however giving much improved results on both the training and test set of megadepth).

Are there some gotchas here which I have not considered?

**Comment by Parskatt** [comment]
Update: I think my issues was actually caused by d2-net caching results from a previous run I did.

**Comment by Parskatt** [comment]
Now I'm getting results that are comparable to the pretrained model.
While I have this thread open. Could you provide what settings you used for the pretrained model? Was it exactly the settings you provide in the configs in this repo?

**Comment by qianqianwang68** [comment]
hi, most of the current settings should be the same as the settings for the pretrained model.
But unfortunately, I'm not able to provide the settings for the pretrained model because the released code has changed and it is different from the code that I ran to get the pretrained model. But you should still be able to get a similar performance if you train your own model using this code.

**Comment by Parskatt** [comment]
Thanks for the reply.
I'll close the issue.

**Comment by jeannotes** [comment]
@Parskatt hiw many epochs ho you train it, and what is your machine setup? seems require lots of memory

**Comment by Parskatt** [comment]
@jeannotes I used the provided config, ie 200000 iterations. If there are issues with memory, perhaps use fewer SIFT points?
